ex8.py

```diff
@@ -0,0 +1,26 @@
+import numpy
+import numpy as np
+
+
+def replace_number(matrix: numpy.ndarray, number: int, action: chr) -> numpy.ndarray:
```

Comment:
There is no char type - 
https://stackoverflow.com/questions/47310929/does-python-support-character-type